packages/common/src/compiler/exploreCompiler.ts

```diff
@@ -21,7 +21,7 @@ import { renderFilterRuleSql } from './filtersCompiler';
 
 // exclude lightdash prefix from variable pattern
 export const lightdashVariablePattern =
-    /\$\{((?!lightdash|ld\b)[a-zA-Z0-9_.]+)\}/g;
+    /\$\{((?!(lightdash|ld)\.)[a-zA-Z0-9_.]+)\}/g;
```

Comment:
Does it work with `table_lightdash.something` ?

Reply:
yes, that works, I've added a test for it